README.md

```diff
@@ -240,6 +240,26 @@ yarn run test
 
 For more testing tips, please check out [docs/running_tests](docs/running_tests.md).
 
+
+## Profiling and local production testing
+
+If you want to test the performance of your changes, you can start up a local server
+with settings closer to a production environment like so:
+
+```bash
+yarn run localprodserver
```

Comment:
This plus `whitenoise` allows gunicorn to serve static assets right?

Reply:
Yup!